core/metrics/src/near_gauge.rs

```diff
@@ -0,0 +1,70 @@
+use crate::IntCounter;
+use prometheus::core::{AtomicI64, AtomicU64, GenericCounter, GenericGauge};
+use prometheus::{Histogram, HistogramOpts, HistogramTimer, IntGauge, Opts};
+
+pub struct NearIntGauge {
+    gauge: GenericGauge<AtomicI64>,
+}
+
+impl NearIntGauge {
+    pub fn new(name: &'static str, help: &'static str) -> Self {
+        let opts = Opts::new(name, help);
+        let gauge = IntGauge::with_opts(opts).unwrap();
+        prometheus::register(Box::new(gauge.clone())).unwrap();
```

Comment:
:thinking: Do we actually want to panic if we can't create a gauge? If that's true, I don't think we need our own wrapper, and can write something like this instead: 

```
    pub static ref PEER_CONNECTIONS_TOTAL: IntGauge =
        try_create_int_gauge("near_peer_connections_total", "Number of connected peers").unwrap();
```

Reply:
@matklad According to the documentation it should fail only if there is a name conflict. I think it's better to panic rather than have those metrics not work silently.

Reply:
@matklad Can you take a look again?

Reply:
@matklad We can do this instead; https://github.com/near/nearcore/pull/5303